DEVELOPERS.md

```diff
@@ -0,0 +1,20 @@
+# Information for Developers
+
+## `Images` Folder
+
+In the `Images` folder, you'll find PDFs of all the graphical assets as well as the `.afdesign` files from the program I created them with, Affinity Designer. We use [Font Awesome](https://fontawesome.com) to ensure a consistent design language across the website and app.
+
+## `Issue Manager` Folder
+
+The `Issue Manager` folder contains the Xcode project and all its associated files.
+
+- Dependencies are managed using Carthage; you can run `carthage bootstrap` in order to fetch the versions the last commit used.
+- The `Libraries` folder contains tiny libraries that are so small we decided to just copy them over and manage them ourselves, as well as adjusting some of the code.
+- The `Back End` folder contains the (platform-agnostic) back-end code which the front end is built around.
+	- `API Objects` contains all the objects defined in the API, encoded/decoded in communicating with the server.
+	- `Requests` contains the concrete representations of the API endpoints, as well as exposing mechanisms to use them.
+	- `Helpers` contains almost exclusively extensions of existing types to simplify code using them elsewhere.
```

Comment:
... and what else is in there? you may want to consider creating a folder `Extensions` for the extensions; and possibly then you can be more specific what the `Helper` folder is about.